packages/apps/docs/src/scripts/remarkHeadersToProps.mjs

```diff
@@ -38,34 +54,21 @@ const remarkHeadersToProps = () => {
       },
     ];
 
-    headers.forEach((item, index) => {
+    headers.forEach((item) => {
       const parent = lastHeading(startArray[0], item);
-      const parentTitle = parent.title ?? '';
+
+      cleanupHeading(item);
 
       // we dont want h1 tags in the aside menu
       if (item.depth === 1) {
         return;
       }
 
-      // Heading3 component needs index and parent title as a props
-      // to generate the unique anchor link to match with the sidebar menu
-      // for any special characters
-      item.data = {
```

Comment:
was needed for hashes of headers.
But this was refactored 2 months ago. not needed anymore